src/util/DateUtils.ts

```diff
@@ -26,14 +26,9 @@ export class DateUtils {
      * Converts given value into date string in a "YYYY-MM-DD" format.
      */
     static mixedDateToDateString(value: string | Date): string {
-        if (value instanceof Date)
-            return (
-                this.formatZerolessValue(value.getFullYear(), 4) +
-                "-" +
-                this.formatZerolessValue(value.getMonth() + 1) +
-                "-" +
-                this.formatZerolessValue(value.getDate())
-            )
+        if (value instanceof Date) {
+            return value.toISOString().split("T")[0]
```

Comment:
are you sure it's the same? 😨

Reply:
It is not the same, my change puts the time in UTC whereas the the before change puts the timezone in whatever timezone the server is on (which is why a test case was failing on my machine locally). I updated this because I assumed that the test case was golden but after reading through more of the source code from this comment, I realized that it may be purposed to format the data into the current time zone. I think that the test case that failed locally should be updated rather than this code. I have reverted this and am leaving that update to the test case outside of the scope of this feature.